main.css

```diff
@@ -0,0 +1,90 @@
+.container {
+  display: flex;
+}
+
+.container > nav {
+  color: #6070ff;
+  margin: 10px;
+  text-align: center;
+  line-height: 75px;
+  font-size: 30px;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: bold;
+}
+
+.nav-item1 {
+  flex-grow: 1;
+}
+
+.nav-item2 {
+  flex-grow: 8;
+}
+
+.nav-item3 {
+  flex-grow: 1;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  height: 180vh;
+  background-image: url("images/background.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.box1 {
+  color: #172b4d;
+  justify-content: center;
+  width: 327px;
+  height: 150px;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
+
+.box2 {
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  justify-content: center;
+  width: 327px;
+  height: 146px;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.box3 {
+  color: #7f8cff;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+}
+
+.box4 {
+  display: flex;
+}
+
+.social-media {
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  width: 30%;
```

Comment:
- [ ] Regarding the styling  of the these social media links, I think it would be nice to use separate them with padding or margin instead of using `width: 30%`.